test/get-own-property-symbols.js

```diff
@@ -318,5 +318,31 @@ wru.test([
       Point.prototype[Symbol.toStringTag] = 'Point';
       wru.assert(Object.prototype.toString.call(new Point) === '[object Point]');
     }
+  }, {
+    name: 'keyFor throws on non symbol',
+    test: function () {
+      var didTrow = false;
+      try {
+        Symbol.keyFor('not a Syumbol');
+      } catch(yay) {
+        didTrow = true;
+      }
+      wru.assert(didTrow, 'non symbols cannot be passed to keyFor');
```

Comment:
wru has an inconvenient signature where the comment that describes what you are testing, if present, is the first parameter, not the last. I think `wru` came out even before `console.assert` so that's how it is, you should invert the check